app/pb/api.proto

```diff
@@ -189,7 +189,7 @@ message User {
   double lng = 34;
   double radius = 35; // meters
 
-  double verification = 5;
+  double verification = 5;  // 1.0 if phone number verified, 0.0 otherwise
```

Comment:
I think this value is an algorithmic value between 0 and 1, of which phone number verification is just a part?

Reply:
Sounds good. Will adjust the comment when we implement other verification methods.